base/checked.jl

```diff
@@ -2,6 +2,13 @@
 
 # Support for checked integer arithmetic
 
+"""
+    Checked
+The Checked module provides arithmetic functions for the built-in signed and unsigned
+Integer types which throw an error when an overflow occurs. They are named like `checked_sub`,
+`checked_div`, etc. In addition, `add_with_overflow, sub_with_overflow, mul_with_overflow`
```

Comment:
I think each function should have its own "code style" environment, like
```
`add_with_overflow`, `sub_with_overflow`
```
etc.